install/prerequisites.adoc

```diff
@@ -149,6 +149,10 @@ The size of storage depends on workload, the number of containers, the
 size of the running containers, and the containers' storage requirements.
 You must also configure storage to run containerized etcd.
 
+It is highly recommended that you use etcd with storage that handles small
+read/write operations quickly, such as SSD. Ceph, NFS, and spinning disks are
```

Comment:
nit: `handles serial writes (fsync) quickly`. Most issues with disk are related to the speed at which fsync can persist state to disk and return.